src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig

```diff
@@ -71,31 +71,31 @@
             <p>No messages have been collected.</p>
         </div>
     {% else %}
-
-    <div class="sf-tabs message-bus">
-        <div class="tab">
-            {% set messages = collector.messages %}
-            {% set exceptionsCount = collector.exceptionsCount %}
-            <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount == messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
-
-            <div class="tab-content">
-                <p class="text-muted">Ordered list of dispatched messages across all your buses</p>
-                {{ helper.render_bus_messages(messages, true) }}
+        <div class="sf-tabs message-bus">
+            <div class="tab">
+                {% set messages = collector.messages %}
+                {% set exceptionsCount = collector.exceptionsCount %}
+                <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount == messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
```

Comment:
Taking advantage of the fact that you are refactoring why you do not use a strict comparison?
```suggestion
                <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount === messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
```

Reply:
Twig doesn't have a `===` operator.

Reply:
haha ... ups ... I missed it ... hehe

Reply:
But it has: 
```
exceptionsCount is same as (messages|length)
```
No sure it worth using it here anyway.